src/MySql/custom/New-AzMySqlServer.ps1

```diff
@@ -177,6 +184,15 @@ function New-AzMySqlServer {
               $Parameter.SslEnforcement = [Microsoft.Azure.PowerShell.Cmdlets.MySql.Support.SslEnforcementEnum]::Enable
           }
 
+          if ($PSBoundParameters.ContainsKey('MinimalTlsVersion')) {
+              $Parameter.MinimalTlsVersion = $PSBoundParameters['MinimalTlsVersion']
+              $null = $PSBoundParameters.Remove('MinimalTlsVersion')
+          }
+          else
+          {
+              $Parameter.MinimalTlsVersion = [Microsoft.Azure.PowerShell.Cmdlets.MySql.Support.MinimalTlsVersionEnum]::TlsEnforcementDisabled
```

Comment:
MinimalTlsVersion is optional. Why assign a default value to it?

Reply:
You're right,  The code is unnecessarily, The MinimalTlsVersion default value is TlsEnforcementDisabled. I have removed it.

Reply:
> You're right, The code is unnecessarily, The MinimalTlsVersion default value is TlsEnforcementDisabled. I have removed it.

Please just remove something in "else {} ". Please check the customized change again

Reply:
I have only removed "else {} ". 
Thanks